js/apps/account-ui/src/oid4vci/Oid4Vci.tsx

```diff
@@ -0,0 +1,204 @@
+import {
+    Select,
+    SelectList,
+    SelectOption,
+    PageSectionVariants,
+    PageSection,
+    ActionList,
+    ActionListItem,
+    List,
+    ListItem,
+    MenuToggleElement,
+    MenuToggle
+} from '@patternfly/react-core';
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useAlerts, useEnvironment } from "@keycloak/keycloak-ui-shared";
+import { usePromise } from "../utils/usePromise";
+import { Page } from "../components/page/Page";
+import { parseResponse } from "../api/parse-response";
+import { token } from '../api/request';
+
+type CredentialsIssuer = {
+  credential_issuer: string;
+  credential_endpoint: string;
+  authorization_servers: string[]; 
+  credential_configurations_supported: Map<string, SupportedCredentialConfiguration>
+}
+
+type SupportedCredentialConfiguration = {
+  id: string,
+  format: string,
+  scope: string
+}
+
+type VCState = {
```

Comment:
Internalized state for a component should be kept in separate `useState()` entries, rather than a single massive object with all members, these should be extracted, for example:

```ts
const [offerQRVisible, setOfferQRVisible] = useState(false);
const [qrCode, setQrCode] = useState("");
// Etc...
```

Reply:
Fixed in https://github.com/keycloak/keycloak/commit/26857215ec6f6053560781e6ce9849202a601bee